Sources/Observing/ListMonitor.swift

```diff
@@ -677,7 +677,7 @@ public final class ListMonitor<T: NSManagedObject>: Hashable {
     
     internal func downcast() -> ListMonitor<NSManagedObject> {
         
-        return unsafeBitCast(self, to: ListMonitor<NSManagedObject>.self)
+        return self as Any as! ListMonitor<NSManagedObject>
```

Comment:
@ruslanskorb The last `as!` cast crashes for me. I don't think this would work because Swift generic types are invariant.